src/components/VideoAsset.vue

```diff
@@ -15,28 +15,43 @@
     :should-wrap="!!deviceFrame"
     :device="deviceFrame"
   >
-    <video
-      ref="video"
-      :controls="showsControls"
-      :data-orientation="orientation"
-      :autoplay="autoplays"
-      :poster="normalisedPosterPath"
-      :muted="muted"
-      :width="optimalWidth"
-      playsinline
-      @loadedmetadata="setOrientation"
-      @playing="$emit('playing')"
-      @pause="$emit('pause')"
-      @ended="$emit('ended')"
-    >
-      <!--
-        Many browsers do not support the `media` attribute for `<source>` tags
-        within a video specifically, so this implementation for dark theme assets
-        is handled with JavaScript media query listeners unlike the `<source>`
-        based implementation being used for image assets.
-      -->
-      <source :src="normalizePath(videoAttributes.url)">
-    </video>
+    <div>
+      <span
+        v-if="alt"
+        :id="alt ? altTextId : null"
+        class="visuallyhidden"
+      >
+        {{ $t('formats.colon', {
+          content: $t('video.description')
+        }) + alt }}
+      </span>
+      <video
+        ref="video"
+        :id="id"
```

Comment:
Just reminding you that when merging `main` back into this, you may get a small conflict due to my most recent PR—just remember to make sure that the `:key="videoAttributes.url"` line persists when resolving it—thanks!